jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -13,6 +13,7 @@ String stash_manifest_name
 String stash_manifest_path
 String TESTS
```

Comment:
like a shell/python script, can we add a comment of what the global parameters mean, what's its value range?